src/sources.cpp

```diff
@@ -23,6 +23,13 @@
 #include "meep.hpp"
 #include "meep_internals.hpp"
 
+// For sine-integral function used in the band-source
+#define HAVE_LIBGSL_EXPINT
```

Comment:
Why are you unconditionally defining this rather than modifying `configure.ac` to check for it?